src/chains/ethereum/ethereum/src/wallet.ts

```diff
@@ -246,13 +246,13 @@ export default class Wallet {
         }
       }
     } else {
-      const numerOfAccounts = options.totalAccounts;
-      if (numerOfAccounts) {
-        accounts = Array(numerOfAccounts);
+      const numberOfAccounts = options.totalAccounts;
+      if (numberOfAccounts != null) {
```

Comment:
Just an idea; if you used `numberOfAccounts == null` then you could have less indentation.